src/components/page-container/PageContainer.jsx

```diff
@@ -31,23 +31,23 @@ export default function PageContainer(props) {
           org: '',
           number: '',
           title: '',
-          isStaff: false,
+          originalUserIsStaff: false,
           tabs: [],
           cohorts: [],
         });
         return;
       }
 
       const {
-        org, number, title, tabs, is_staff: isStaff,
```

Comment:
While were changing this stuff, can we do this camel case conversion in `api.js` instead of the component itself? theres a `camelCaseObject` function you can import in `@edx/frontend-platform`.

Reply:
I'll look into this. Thanks!